ct-aa/dotTheory---TypeScript
# JS-972: Added example activities

# Related Tickets
[JS-972](https://codeandtheory.atlassian.net/browse/JS-972)

# Description
- Added 3 additional folders for the sample projects. (lodash-and-types, class-validator, class-transformer).

# Validation
- Open the terminal.
- Change the directory to each of the new folders for the sample projects.
- Run "npm install".
- Run "npm start".
- Open the browser and check the local host.
- Inspect the browser.
- The result for each sample project should be displayed in the console.

lodash-and-types
<img width="1787" alt="Screen Shot 2024-01-23 at 5 04 23 PM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/bb8fca3c-f314-41e9-93c4-67d2aa68a66e">

class-validator
<img width="1792" alt="Screen Shot 2024-01-23 at 5 05 24 PM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/335edaf2-bd66-4646-98d6-f52168a0359f">

class-transformer
<img width="1792" alt="Screen Shot 2024-01-23 at 5 14 38 PM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/e584b02f-5539-4ab3-b06a-040d0741b883">

# Risks
N/A
